pytorch/inference/buildspec-1-13-sm.yml

```diff
@@ -32,38 +32,6 @@ context:
       target: deep_learning_container.py
 
 images:
-  BuildEC2CPUPTInferencePy3DockerImage:
```

Comment:
I see there is already a buildspec for 1.13 ec2, but can you make sure it is updated with latest contents from this file? At least TS version is lagging in the existing one

Reply:
Thanks, yes, modified it in the latest commit 👍

Reply:
nice most recent change lgtm